tests/tt_eager/python_api_testing/unit_testing/test_moreh_adam.py

```diff
@@ -0,0 +1,163 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+import tt_lib as ttl
+import pytest
+from models.utility_functions import (
+    skip_for_wormhole_b0,
+    comp_allclose_and_pcc,
+    comp_pcc,
+)
+from loguru import logger
+
+
+@pytest.mark.parametrize(
+    "shape",
+    (
+        (1, 1, 32, 32),  # single
+        (12, 6, 64, 64),  # multi tile
+    ),
+)
+@pytest.mark.parametrize("lr", [1e-3, 1e-2, 1e-1])
+@pytest.mark.parametrize("betas", ((0.9, 0.999), (0.5, 0.555)))
+@pytest.mark.parametrize("eps", [1e-06, 1e-08])
+@pytest.mark.parametrize("weight_decay", [0.0, 0.1, 0.3])
+@pytest.mark.parametrize("amsgrad", [True, False])
```

Comment:
```suggestion
@pytest.mark.parametrize("lr", [0.0, 1e-1])
@pytest.mark.parametrize("betas", ((0.9, 0.999), (0.5, 0.555)))
@pytest.mark.parametrize("eps", [1e-06, 1e-08])
@pytest.mark.parametrize("weight_decay", [0.0, 0.3])
@pytest.mark.parametrize("amsgrad", [True, False])
```
I don't think we need this much lr and weight_decay test parameters. 
because there is no speical change like 'if statement branch change' depend on lr and weight_decay.

this change can reduce test case 9 -> 4.

Reply:
applied the feedback